src/Raven.Server/NotificationCenter/Handlers/ClusterDashboardHandler.cs

```diff
@@ -61,7 +61,7 @@ public async Task Watch()
 
                         var currentCert = GetCurrentCertificate();
 
-                        using (var connection = new ProxyWebSocketConnection(webSocket, remoteNodeUrl, $"/admin/cluster-dashboard/remote/watch?thumbprint={currentCert?.Thumbprint}", ServerStore.ContextPool, ServerStore.ServerShutdown))
+                        using (var connection = new ProxyWebSocketConnection(webSocket, Server.ServerStore.NodeTag, remoteNodeUrl, $"/admin/cluster-dashboard/remote/watch?thumbprint={currentCert?.Thumbprint}", ServerStore.ContextPool, ServerStore.ServerShutdown))
```

Comment:
why?

Reply:
```GetTcpInfoAsync``` now sends the source node tag for checking in the other side (dest) if it's rehab in it's topology.

Reply:
Removed